src/wp-admin/includes/template.php

```diff
@@ -955,37 +955,43 @@ function add_meta_box( $id, $title, $callback, $screen = null, $context = 'advan
 			if ( !isset($wp_meta_boxes[$page][$a_context][$a_priority][$id]) )
 				continue;
 
-			// If a core box was previously added or removed by a plugin, don't add.
-			if ( 'core' == $priority ) {
-				// If core box previously deleted, don't add
-				if ( false === $wp_meta_boxes[$page][$a_context][$a_priority][$id] )
-					return;
+			// If a core box was previously removed, don't add.
+			if ( ( 'core' === $priority || 'sorted' === $priority )
+				&& false === $wp_meta_boxes[ $page ][ $a_context ][ $a_priority ][ $id ]
+			) {
```

Comment:
This indentation is wacky and I'm surprised it was committed to WP this way:

```suggestion
			if (
				( 'core' === $priority || 'sorted' === $priority ) &&
				false === $wp_meta_boxes[ $page ][ $a_context ][ $a_priority ][ $id ]
			) {
```